### ISSUE TITLE
Accessibility: Group radio buttons for ORCID

### ISSUE THREAD

**Issue by astridu** [sul-dlss/happy-heron#1990]
These 2 radio buttons need to be grouped together.
![image](https://user-images.githubusercontent.com/42216982/130171438-57a5afac-8d9f-424f-be75-1f3d4e4e7952.png)
Form elements are not grouped
 3.3.2 Labels or Instructions
When a form contains elements that are logically connected to each other it is necessary to group them.

The form contains check boxes and/or radio buttons that are not grouped and named, using a 'fieldset' with a descriptive 'legend' or the WAI-ARIA attribute role="group" (for check boxes) or role="radiogroup" (for radio buttons) with a descriptive 'aria-labelledby' or 'aria-label'.

**Comment by jcoyne** [comment]
@astridu radiogroups are required to have an aria-labelledby / aria-label.  What is the label for this group of radio buttons?  Do you want this label to display somewhere on the form?  How do you want it to look?

**Comment by astridu** [comment]
Is it possible to use 'Role term' as the label and have it appear over the whole section like this? 
<img width="626" alt="image" src="https://user-images.githubusercontent.com/42216982/186069886-47663083-2b61-4096-a7f4-e9b779995e80.png">

**Comment by astridu** [comment]
or does it need to be directly over the radio buttons?

**Comment by jcoyne** [comment]
"Role term" is the label for the drop down (where we see "Author" in the mockup).  I  think that the label for the radio group would have to be distinct from the label for that field.

**Comment by astridu** [comment]
@amyehodge The only thing generic enough that I can think of to label that section is 'Details'. What do you think?

**Comment by amyehodge** [comment]
@astridu @jcoyne "Details" as the label is fine.

**Comment by jcoyne** [comment]
@astridu where do you want to put that label?

**Comment by astridu** [comment]
Can we do it like this? 
<img width="1150" alt="image" src="https://user-images.githubusercontent.com/42216982/186546815-81744cdb-a672-46dd-a335-76d7e4e80a29.png">

**Comment by jcoyne** [comment]
Yes, we could do that. Do we still want the "is" there?  It seems awkward now.  How do you read that "Author is details"?

**Comment by astridu** [comment]
I see your point. We should remove the 'is'. Thank you!
